src/Tickets/Commerce/Status/Pending.php

```diff
@@ -60,8 +60,37 @@ public function get_name() {
 	 * {@inheritdoc}
 	 */
 	public function can_apply_to( $order, $new_status ) {
-		$status = parent::can_apply_to( $order, $new_status );
+		return $this->can_transition_from_pending_to_status( parent::can_apply_to( $order, $new_status ), $order );
+	}
 
+	/**
+	 * Whether a Status Interface can be applied on top of another Status Interface.
+	 *
+	 * @since TBD
+	 *
+	 * @param self $new_status     The new status.
+	 * @param ?int $order_id       Which order we are testing against.
+	 *
+	 * @return bool|WP_Error Whether the new status can be applied to the current status.
+	 */
+	public function can_transition_to_status( $new_status, $order_id = null ) {
+		return $this->can_transition_from_pending_to_status(
```

Comment:
Why put this in a second function? It could all live in here, overriding the parent value.

Reply:
undid all of that